CONTRIBUTING.md

```diff
@@ -28,27 +28,20 @@ We use `pytest` for our unit testing, making use of `parametrized` to inject cas
 
 These are planned once the SDK has been stabilized and a Flagd provider implemented. At that point, we will utilize the [gherkin integration tests](https://github.com/open-feature/test-harness/blob/main/features/evaluation.feature) to validate against a live, seeded Flagd instance.
 
-### Packaging
-
-We publish to the PyPI repository, where you can find each individual package:
-
-- [openfeature-provider-flagd](https://pypi.org/project/openfeature-provider-flagd/)
-- [openfeature-hooks-opentelemetry](https://pypi.org/project/openfeature-hooks-opentelemetry/)
-
 ## Pull Request
 
 All contributions to the OpenFeature project are welcome via GitHub pull requests.
 
 To create a new PR, you will need to first fork the GitHub repository and clone upstream.
 
 ```bash
-git clone https://github.com/open-feature/python-sdk-contrib.git openfeature-python-sdk-contrib
+git clone https://github.com/open-feature/python-sdk-contrib.git
```

Comment:
I think the original rename here is a nice touch, we do it in the CONTRIBUTING.md file of the python-sdk and other OpenFeature repos too. Can we keep it?

Reply:
reverted in [bc93bcc](https://github.com/open-feature/python-sdk-contrib/pull/38/commits/bc93bcc79ab1cc676a2c1265dd5f0e715eaa7d64)